css/layout.css

```diff
@@ -16,6 +16,8 @@
 .column-full,
 .column-half {
   width: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
You shouldn't change every element with this class like this. Now every `column-half` or `column-full` element will always be a column flexbox, and that is likely not the goal. You should apply this to your own custom class, and add that class to the same element you want to have that styling.